packages/amplify-provider-awscloudformation/lib/system-config-manager.js

```diff
@@ -56,6 +56,7 @@ function setProfile(awsConfig, profileName) {
 
   fs.writeFileSync(credentialsFilePath, ini.stringify(credentials));
   fs.writeFileSync(configFilePath, ini.stringify(config));
+  ensureCredentialsFilePermission();
```

Comment:
Is the file created on line 57?
Then move ensureCredentialsFilePermission() to 58.